### PR TITLE
fix(docker): fix dockerfile not building the SSR version properly

### PR DESCRIPTION
**Pull request by MrTimscampi**
For some reason, npm wasn't passing down the standalone flag to nuxt-build. As a result, dependencies weren't bundled anymore, leading to errors when trying to run it.

This adds a dedicated standalone build command to avoid having to pass down flags.

### CODE REVIEW

**Review by ThibaultNocchi — APPROVED** [review]
LGTM

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ jobs:
         run: npm ci --no-audit
 
       - name: Build client
-        run: npm run build --production --standalone
+        run: npm run build:standalone
```
Comment (ferferga) [inline .github/workflows/docker.yml:47]:
```suggestion
        run: "npm run build:standalone"
```

File: .github/workflows/docker-release.yml [hunk 2]
```diff
@@ -37,7 +37,7 @@ jobs:
         run: npm ci --no-audit
 
       - name: Build client
-        run: npm run build --production --standalone
+        run: npm run build:standalone
```
Comment (ferferga) [inline .github/workflows/docker-release.yml:40]:
```suggestion
        run: "npm run build:standalone"
```

**Review by ferferga — CHANGES_REQUESTED** [review]
This won't work unless you quote it.

**Review by ferferga — APPROVED** [review]

**Pull request merged**